vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/internal/CardInputField.kt

```diff
@@ -184,14 +184,24 @@ internal class CardInputField(context: Context) : BaseInputField(context),
     internal fun setOutputNumberDivider(divider: String?) {
         when {
             divider.isNullOrEmpty() -> outputDivider = EMPTY_CHAR
+            arrayOf("#", "\\").contains(divider) -> printWarning(
+                TAG,
+                R.string.error_output_divider_mask
+            ).also {
+                outputDivider = EMPTY_CHAR
```

Comment:
@Endorf I don't know if this correct, but looks like in the same situation in `SSnInputField` here we would set `SPACE`, just check it, please.

Reply:
Comparatively, In the `VGSCardNumberEditText` field we have different default behavior. 
By default, we should remove all dividers before submit.